### PR TITLE
Add change-api-status command support

### PR DESCRIPTION
**Pull request by npamudika**
## Purpose
This PR contains the implementation to add support for change-api-status command in APICTL.

Please refer https://github.com/wso2/product-apim-tooling/issues/278 for more details.

## Documentation
https://github.com/wso2/product-apim-tooling/issues/290

## Samples
Sample API state change commands are like:
```
apictl change-api-status -a Publish -n TwitterAPI -v 1.0.0 -r admin -e dev
apictl change-api-status -a Publish -n FacebookAPI -v 2.1.0 -e production
```

## Test environment
Tested in Linux OS

### CODE REVIEW

File: import-export-cli/docs/apictl_change-api-status.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+```
+apictl change-api-status (--action <action-of-the-api-state-change> --name <name-of-the-api> --version <version-of-the-api> --provider <provider-of-the-api> --environment <environment-from-which-the-api-state-should-be-changed>) [flags]
+```
+
+### Examples
+
+```
+apictl change-api-status -a Publish -n TwitterAPI -v 1.0.0 -r admin -e dev
+apictl change-api-status -a Publish -n FacebookAPI -v 2.1.0 -e production
+NOTE: All the 4 flags (--action (-a), --name (-n), --version (-v), and --environment (-e)) are mandatory.
```
Comment (malinthaprasan) [inline import-export-cli/docs/apictl_change-api-status.md:18]:
```suggestion
NOTE: The 4 flags (--action (-a), --name (-n), --version (-v), and --environment (-e)) are mandatory.
```

File: import-export-cli/cmd/deleteAPI.go [hunk 2]
```diff
@@ -43,8 +43,7 @@ const deleteAPICmdLongDesc = "Delete an API from an environment"
 
 const deleteAPICmdExamples = utils.ProjectName + ` ` + deleteAPICmdLiteral + ` -n TwitterAPI -v 1.0.0 -r admin -e dev
 ` + utils.ProjectName + ` ` + deleteAPICmdLiteral + ` -n FacebookAPI -v 2.1.0 -e production
-NOTE: All the 3 flags (--name (-n), --version (-v), and --environment (-e)) are mandatory.
-If the --provider (-r) is not specified, the logged-in user will be considered as the provider.`
+NOTE: All the 3 flags (--name (-n), --version (-v), and --environment (-e)) are mandatory.`
```
Comment (malinthaprasan) [inline import-export-cli/cmd/deleteAPI.go:46]:
```suggestion
NOTE: The 3 flags: --name (-n), --version (-v), and --environment (-e) are mandatory.`
```

File: import-export-cli/cmd/changeAPIStatus.go [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+var apiStateChangeAction string
+
+// ChangeAPIStatus command related usage info
+const changeAPIStatusCmdLiteral = "change-api-status"
+const changeAPIStatusCmdShortDesc = "Change API Status"
+const changeAPIStatusCmdLongDesc = "Change the lifecycle status of an API in an environment"
+
+const changeAPIStatusCmdExamples = utils.ProjectName + ` ` + changeAPIStatusCmdLiteral + ` -a Publish -n TwitterAPI -v 1.0.0 -r admin -e dev
+` + utils.ProjectName + ` ` + changeAPIStatusCmdLiteral + ` -a Publish -n FacebookAPI -v 2.1.0 -e production
+NOTE: All the 4 flags (--action (-a), --name (-n), --version (-v), and --environment (-e)) are mandatory.`
```
Comment (malinthaprasan) [inline import-export-cli/cmd/changeAPIStatus.go:46]:
```suggestion
NOTE: The 4 flags: --action (-a), --name (-n), --version (-v), and --environment (-e) are mandatory.`
```

File: import-export-cli/docs/apictl_change-api-status.md (hunk 1 above)
Comment (npamudika) [reply import-export-cli/docs/apictl_change-api-status.md:18]: +1

File: import-export-cli/cmd/deleteAPI.go (hunk 2 above)
Comment (npamudika) [reply import-export-cli/cmd/deleteAPI.go:46]: +1

File: import-export-cli/docs/apictl_change-api-status.md (hunk 1 above)
Comment (npamudika) [reply import-export-cli/docs/apictl_change-api-status.md:18]: Fixed via https://github.com/wso2/product-apim-tooling/pull/295

File: import-export-cli/cmd/deleteAPI.go (hunk 2 above)
Comment (npamudika) [reply import-export-cli/cmd/deleteAPI.go:46]: Fixed via https://github.com/wso2/product-apim-tooling/pull/295

File: import-export-cli/cmd/changeAPIStatus.go (hunk 3 above)
Comment (npamudika) [reply import-export-cli/cmd/changeAPIStatus.go:46]: Fixed via https://github.com/wso2/product-apim-tooling/pull/295

**Pull request merged**